samples/bluetooth/central_iso/README.rst

```diff
@@ -26,4 +24,21 @@ Building and Running
 This sample can be found under :zephyr_file:`samples/bluetooth/central_iso` in
 the Zephyr tree.
 
-See :ref:`bluetooth samples section <bluetooth-samples>` for details.
+1. Start the application.
+   In the terminal window, check that it is scanning for other devices.
+
+      Bluetooth initialized
+      Scanning successfully started
+      Device found: D3:3A:5D:F5:73:33 (random) (RSSI -78)
+      Device found: 70:7B:F4:2B:76:AD (random) (RSSI -68)
+      Device found: 65:CF:20:0D:CB:9D (random) (RSSI -82)
+
+2. Observe that the device connects.
+
+      Connected: FA:BB:79:57:D6:45 (random)
```

Comment:
Slightly odd that the `Connected` device isn't part of the devices found in step 1